test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -191,21 +298,24 @@ define([
         });
     });
 
-    describe('the job status table', () => {
+    describe('The job status table', () => {
         let container, jobStatusTableInstance;
         afterEach(() => {
             TestUtil.clearRuntime();
+            if (container) {
```

Comment:
I was getting some mischief occurring with tests, so this is to make sure that everything is cleared up, including any DataTables stuff that might be lying around.

Reply:
I've run into the same thing. DataTables can be mischievous!